language/anope.fr_FR.po

```diff
@@ -9105,5 +9076,66 @@ msgstr "{pseudo | canal}"
 msgid "{nick | channel} memo-text"
 msgstr "{pseudo | canal} texte"
 
+#~ msgid "channel ADD mask"
```

Comment:
Can you delete the old unused strings from the end of the file?

Reply:
done